assets/stylesheets/bootstrap.scss

```diff
@@ -5,3 +5,4 @@
 @import 'base/modules';
 @import 'components/modules';
 @import 'themes/modules';
+@import 'utilities/modules';
```

Comment:
Good question. What do you think, @zarahzachz?

I created the new directory following [the article you sent me yesterday](https://scotch.io/tutorials/aesthetic-sass-1-architecture-and-style-organization). I'd be happy to take on refactoring the current utilities into their own modules, unless you think it makes more sense to keep them in a single file?